### ISSUE TITLE
Alpha value seems to grow with epochs

### ISSUE THREAD

**Issue by noofaq** [cgorman/tensorflow-som#7]
I have noticed that my initial learning rate value didn't have any meaning for results of the training. After some analysis I have found a part of the code which has raised my eyebrows:
https://github.com/cgorman/tensorflow-som/blob/e69ed79357d9d7d4ac846630940a0f57551caf41/tf_som.py#L255-L260

It seems that alpha value grows with each iteration linearly from 0 to 1 regardless of the initial_learning_rate value set in the constructor. I expect that alpha value should decay overtime (if I understand its meaning correctly) starting from the learning rate going towards 0.

Radius value decreases from initial value to 1 in last iteration - this is correct so no issue here.

**Comment by cgorman** [comment]
Oh jeez you're right. Not sure how I missed that one. I must have just copied and pasted the radius one and forgot to change it. I'll fix it tonight, thank you for noticing!

**Comment by cgorman** [comment]
Ok I fixed it and it seems to be working now. Thank you again for noticing and letting me know.
